marklogic-data-hub/src/main/java/com/marklogic/hub/flow/impl/FlowImpl.java

```diff
@@ -208,7 +208,7 @@ public String serialize() {
             return finalWriter.toString().replaceFirst("<!--", "\n<!--").replaceFirst("-->", "-->\n");
         }
         catch(Exception e) {
```

Comment:
Throwing just a string message for any Exception seems very greedy and like it might obscure other issues.  Are you sure about this?  You can always add another block.